src/main/cs/RLBotDotNet/RLBotDotNet/Interface/RLBotInterface.cs

```diff
@@ -164,7 +173,17 @@ public static void SetGameStatePacket(GameStatePacket gameState)
         public static void RenderPacket(RenderPacket finishedRender)
         {
             byte[] bytes = finishedRender.Bytes;
-            RenderGroup(bytes, bytes.Length);
+            int status = RenderGroup(bytes, bytes.Length);
+
+            if (status > 0)
+                throw NewRLBotCoreException((RLBotCoreStatus)status);
+        }
+
+
+        private static RLBotCoreException NewRLBotCoreException(RLBotCoreStatus status)
```

Comment:
What's your reasoning for making this static helper method instead of another constructor for RLBotCoreException?

Reply:
As far as I'm aware you can only set the Message field of an exception by passing a string to the constructer. So if I would have made a constructor taking a RLBotCoreStatus and then generate a string, there would be no way of setting that Message field.

Reply:
We discussed offline about ways to do this. Since I can't reach you at the moment, I'm going to go ahead and merge this then follow up with another commit.